src/System.ServiceModel.Primitives/ref/System.ServiceModel.Primitives.cs

```diff
@@ -844,6 +844,8 @@ public sealed partial class BinaryMessageEncodingBindingElement : System.Service
         public System.ServiceModel.Channels.CompressionFormat CompressionFormat { get { return default(System.ServiceModel.Channels.CompressionFormat); } set { } }
         [System.ComponentModel.DefaultValueAttribute(2048)]
         public int MaxSessionSize { get { return default(int); } set { } }
+        public int MaxReadPoolSize { get; set; }
```

Comment:
I think both of these need to declare the default return type like this...

`public int MaxReadPoolSize { get { retrun default(int); } set { } }`